### PR TITLE
replaced str.index with enumerate

### PR DESCRIPTION
**Pull request by ali0gamer1**

### CODE REVIEW

File: simple hash function.py [hunk 1]
```diff
@@ -5,11 +5,12 @@
...
+    for i,c in enumerate(strs):
+        seed += ord(c)* i + 42
+        
     random.seed(seed)
     for x in range(character):
         str_hash+=random.choice(string.hexdigits[:16])
     return str_hash
 
-#hash_fun('hi') --> 'f843c001'
+print(hash_fun('hi'))
```
Comment (seyedmahdi4) [inline simple hash function.py]: i run it at python shell :)

File: simple hash function.py [hunk 2]
```diff
@@ -5,11 +5,12 @@
 def hash_fun(strs):
     seed=0
     str_hash=''
-    for s in strs:
-        seed += ord(s)* strs.index(s)+42
+    for i,c in enumerate(strs):
```
Comment (seyedmahdi4) [inline simple hash function.py]: nice ...but I wrote this to make it more understandable for everyone

File: simple hash function.py (hunk 1 above)
Comment (ali0gamer1) [reply simple hash function.py]: pls dont gir alaki

File: simple hash function.py (hunk 2 above)
Comment (ali0gamer1) [reply simple hash function.py]: enumerate چیه مگه که سخت باشه

**Pull request merged**